pkgs/build-support/trivial-builders/default.nix

```diff
@@ -315,6 +315,21 @@ rec {
         "nounset"
         "pipefail"
       ],
```

Comment:
We should probably rename bashOptions to shellOptions.

Maybe it is as easy as this?
```nix
      shellOptions ? [
        "errexit"
        "nounset"
        "pipefail"
      ],
      bashOptions ? shellOptions
```